CMakeLists.txt

```diff
@@ -1252,13 +1253,12 @@ add_library_variant(
     RAM_ADDRESS 0x41000000
     RAM_SIZE 0x1000000
     STACK_SIZE 8K
-    ENABLE_RTTI_EXCEPTIONS OFF
+    ENABLE_RTTI_EXCEPTIONS ON
 )
 add_library_variant(
     aarch64
-    SUFFIX exceptions_rtti
     COMPILE_FLAGS "-march=armv8-a"
-    MULTILIB_FLAGS "--target=aarch64-none-unknown-elf"
+    MULTILIB_FLAGS "--target=aarch64-none-unknown-elf -fno-exceptions -fno-rtti"
```

Comment:
Wouldn't it be better to add this inside `add_library_variant` based on the `ENABLE_RTTI_EXCEPTIONS` setting?
BTW: Maybe it would be useful to also add `-fexceptions` and `-frtti` options, when the extensions are enabled.